tools/mmp/driver.cs

```diff
@@ -1063,32 +1081,11 @@ static int Compile ()
 				Frameworks.Gather (App, platform_assembly, BuildTarget.Frameworks, BuildTarget.WeakFrameworks);
 			}
 
-			try {
-				string [] env = null;
-				if (!IsUnifiedFullSystemFramework)
-					env = new [] { "PKG_CONFIG_PATH", Path.Combine (FrameworkLibDirectory, "pkgconfig") };
-
-				RunCommand (pkg_config, new [] { "--cflags", "mono-2" }, env, cflagsb);
-				RunCommand (pkg_config, new [] { "--variable=libdir", "mono-2" }, env, libdirb);
-				var versionFile = "/Library/Frameworks/Mono.framework/Versions/Current/VERSION";
-				if (File.Exists (versionFile)) {
-					mono_version.Append (File.ReadAllText (versionFile));
-				} else {
-					RunCommand (pkg_config, new [] { "--modversion", "mono-2" }, env, mono_version);
-				}
-			} catch (Win32Exception e) {
-				throw new MonoMacException (5301, true, e, Errors.MM5301);
```

Comment:
that was my hint something could be missing (was actually duped) in the newer code